changelogs/fragments/9483-sensu-deprecation.yml

```diff
@@ -0,0 +1,6 @@
+deprecated_features:
+  - sensu_check - module is deprecated and will be removed in community.general 13.0.0, use collection ``sensu.sensu_go`` instead (https://github.com/ansible-collections/community.general/pull/9483).
```

Comment:
It's a bit ironic that we deprecate the modules now that sensu.sensu_go will get removed from Ansible 12 ;-)